exporter/awss3exporter/config.go

```diff
@@ -109,5 +120,9 @@ func (c *Config) Validate() error {
 			errs = multierr.Append(errs, errors.New("marshaler does not support compression"))
 		}
 	}
+
+	if c.S3Uploader.RetryMode != "" && c.S3Uploader.RetryMode != "standard" && c.S3Uploader.RetryMode != "adaptive" {
+		errs = multierr.Append(errs, errors.New("invalid retry mode, must be either 'standard' or 'adaptive'"))
```

Comment:
can it also be unset?

Reply:
If we stick with the supported retry mode, `standard` and `adaptive` are the only values. However, there indeed is a [aws.NopRetryer](https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/aws#NopRetryer) that effectively disables the retry. I suppose we can accept `nop` as a valid retry mode and set up the `NopRetryer` when the user chooses to use it. WDYT?